config/locales/en.yml

```diff
@@ -430,6 +430,8 @@ en:
       comment:
         none: None
       created_at: Reported
+      forwarded: Forwarded
+      forwarded_unknown: Unknown (reported before the information was recorded)
```

Comment:
Unnecessary detailedness. You can just write "not available" or reuse "n/a" from somewhere if we already have that. Just using `-` is also an option.